sylt-compiler/src/dependency.rs

```diff
@@ -373,6 +383,19 @@ pub(crate) fn initialization_order<'a>(
         for statement in module.statements.iter() {
             use StatementKind::*;
             match &statement.kind {
+                From { imports, .. } => {
+                    let mut ctx = Context { compiler, namespace, variables: Vec::new() };
+                    imports.iter().for_each(|Identifier { name, .. }| {
+                        to_order.insert(
+                            (name.clone(), namespace),
+                            (
+                                statement_dependencies(&mut ctx, statement),
```

Comment:
This is not really true since it means every variable in the from import must be initialized.
We could probably replace this with a read assignable in the other namespace...

Reply:
It shouldn't make much difference